scriptmodules/ports/xrick.sh

```diff
@@ -45,8 +45,8 @@ function configure_xrick() {
 
     # set dispmanx by default on rpi with fkms
     isPlatform "dispmanx" && ! isPlatform "videocore" && setBackend "$md_id" "dispmanx"
-
-    ln -sf "$romdir/ports/xrick/data.zip" "$md_inst/data.zip"
```

Comment:
Is this not needed? I couldn't see any mention to the removal in the commit msg.

Reply:
I'll check, may be an error.

Reply:
Ok, it was an error. The `data.zip` file from the original game (downloadable from http://www.bigorno.net/xrick/download.html) should be symlinked to `$md_inst`.

I restored the original line that did the symlink.